Makefile

```diff
@@ -412,22 +417,30 @@ release-compress:
 	fi
 	cd $(DIST)/release/; for file in `find . -type f -name "*"`; do echo "compressing $${file}" && gxz -k -9 $${file}; done;
 
-npm-check:
-	@hash npm > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
-		echo "Please install Node.js 8.x or greater with npm"; \
+.PHONY: go-check
+go-check:
+	$(eval GO_MISSING := $(shell hash go > /dev/null 2>&1 || echo 1))
+	@if [ "$(GO_MISSING)" = "1" ]; then \
+		echo "Gitea requires Go 1.11 or greater to build. You can get it at https://golang.org/dl/"; \
```

Comment:
```suggestion
		echo "Gitea requires Go 1.13 or greater to build. You can get it at https://golang.org/dl/"; \
```

Reply:
Is it really 1.13? I was going by this comment I found in `drone.yml`:

> image: golang:1.11 # this step is kept as the lowest version of golang that we support

Reply:
It's actually unclear. In Gitea's _"Installation from source"_:

> Note 2: Go version 1.11 or higher is required. However, it is recommended to obtain the same version as our continuous integration, see the advice given in Hacking on Gitea

And in _"Hacking on Gitea"_:

> Note 2: Go version 1.11 or higher is required; however, it is important to note that our continuous integration will check that the formatting of the source code is not changed by gofmt using make fmt-check. Unfortunately, the results of gofmt can differ by the version of go. It is therefore recommended to install the version of go that our continuous integration is running. At the time of writing this is Go version 1.12; however, this can be checked by looking at the master .drone.yml (At the time of writing line 67 is the relevant line - but this may change.)

So, as far as "you need to install golang", I'd go with 1.13 that will cause less confusion. But it's not strictly required for building.

Reply:
That's why I have 'Go 1.11 or greater', I think it's sufficient to just state the minimum.